circle.yml

```diff
@@ -56,14 +56,19 @@ defaults:
 
         ctest --output-on-failure -j $TEST_PARALLEL_JOBS
 
-  store-eth: &store-artifacts
+  store-eth: &store-eth
```

Comment:
Not sure where these changes came from. Let me know if this looks right or if I reverted something.